src/Reflection/ReflectionType.php

```diff
@@ -86,7 +86,7 @@ public function __toString()
                 return 'bool';
             case $this->type instanceof Types\Float_:
                 return 'float';
-            case $this->type instanceof Types\Void:
+            case $this->type instanceof Types\Void_:
```

Comment:
There's probably a test missing for return type void. The Types\Void class does not exist in phpDocumentor anymore.